recipes/jcc/run_test.bat

```diff
@@ -0,0 +1,20 @@
+
+
+
+:: Paths assume java-jdk installed by conda
+set JCC_JDK=%PREFIX%\Library
+set JAVA_HOME=%JCC_JDK%
+set PATH=%JCC_JDK%\jre\bin\server;%JCC_JDK%;%JCC_JDK%\bin;%JCC_JDK%\lib;%JCC_JDK%\include;%PATH%
+set PATH=%JCC_JDK%\jre\bin\server;%JCC_JDK%\jre\bin;%JCC_JDK%\bin;%JCC_JDK%\lib;%JCC_JDK%\include;%PATH%
+set JDK_HOME=%JCC_JDK%
+
+:: set
```

Comment:
JCC needs the PATH (in windows) set to the JDK/JRE directory of the libs. Junk below removed..